WeakAuras/Transmission.lua

```diff
@@ -1697,10 +1697,10 @@ function WeakAuras.Import(inData, target)
       target = targetData
     end
   end
-  WeakAuras.PreAdd(data)
```

Comment:
This might be good to expose publicly, though the name is a bit opaque.

Other addons which want to interface with WeakAuras by adding data might want to ask WeakAuras to fudge the data into something that's actually usable as an aura. Which is what this function does.

Reply:
Okay.